webapp/channels/src/components/admin_console/admin_definition.jsx

```diff
@@ -2447,6 +2447,17 @@ const AdminDefinition = {
                             it.licensedForSku(LicenseSkus.Professional),
                         )),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_TEXT,
+                        key: 'ServiceSettings.RefreshPostStatsRunTime',
+                        label: t('admin.team.refreshPostStatsRunTimeTitle'),
+                        label_default: 'Refresh Post Stats Run Time:',
+                        help_text: t('admin.team.refreshPostStatsRunTimeDescription'),
+                        help_text_default: "The time of day to refresh the user's post stats, including their number of posts and the last record post timestamp.",
```

Comment:
Probably hard to do in this context, but we should make clear to the user that this is 00:00 server time. So it would be great to show somewhere what is the current server time. It may not be the same as the local time of the sysadmin.

Reply:
@abhijit-singh Do you have thoughts on a string we could use here? I'm seeing this for the compliance export job: `Set the start time of the daily scheduled compliance export job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.`

Reply:
Here's my recommendation for the field title and help text:
Title - User Statistics Update Time:
Help text - Set the server time for updating the user post statistics, including each user's total post count and the timestamp of their most recent post. Must be a 24-hour time stamp in the form HH:MM based on the local time of the server.

What do you guys think?

Reply:
Works for me :) @larkox ?

Reply:
LGTM